.github/actions/deploy/action.yaml

```diff
@@ -0,0 +1,66 @@
+name: Deploy
+description: Deploys new Lambda function code
+inputs:
+  aws-access-key-id:
+    description: AWS access key
+    required: true
+  aws-secret-access-key:
+    description: AWS secret key
+    required: true
+  dry-run:
+    description: >-
+      If truthy, the deploy will be a dry run, or will not be attempted at all
+      if `aws-access-key-id` is blank (instead of erroring)
+    required: false
+    default: ""
+  environment:
+    description: Name of the environment to deploy to (dev/prod)
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: ./.github/actions/deps
+
+    - name: Create build folder
+      working-directory: py_gtfs_rt_ingestion
+      run: |
+        poetry build 
+        poetry run pip install --upgrade -t package dist/*.whl 
+        cp -v batch_files.py package/batch_files.py 
+        cp -v ingest.py package/ingest.py
+      shell: bash
```

Comment:
Use poetry to create `dist` and `package` folders. Copy `batch_files.py` and `ingest.py` into packages folder for inclusion in zip archive